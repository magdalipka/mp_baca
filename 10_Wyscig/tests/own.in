1
3 2
a 5 2
b -1 6
c 5 6