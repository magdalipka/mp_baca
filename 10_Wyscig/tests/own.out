a - 0 pkt.
b - 0 pkt.
c - 1 pkt.